doc/managing-certificates-for-application-ingress.md

```diff
@@ -12,21 +12,35 @@ kubectl create secret tls wildcard-apps-example-com-cert \
 
 _Note: This can be either a wildcard cert or a cert for a single domain._
 
-2. Add a new entry under `spec.servers` to the [Istio `Gateway`](https://istio.io/docs/reference/config/networking/gateway/) object in the namespace that your apps are running in. This is `cf-workloads` by default. So, to add configuration for `'*.apps.example.com'` you would need to add this entry to the `spec.servers` array:
-```
-- hosts:
-  # the host from your certificate
-  - '*.apps.example.com'
-  port:
-    name: https-apps-example-com
-    number: 443
-    protocol: HTTPS
-  tls:
-    # secret created in first step
-    credentialName: wildcard-apps-example-com-cert
-    mode: SIMPLE
+2. Add a new entry under `spec.servers` to the [Istio
+   `Gateway`](https://istio.io/docs/reference/config/networking/gateway/) object
+   in the namespace that your apps are running in. This is `cf-workloads` by
+   default. So, to add configuration for `'*.apps.example.com'` you would need
+   to add this entry to the `spec.servers` array. Here is an example [`ytt`](https://get-ytt.io/)
+   overlay that adds it:
+
+```yaml
+#@ load("@ytt:overlay", "overlay")
+
+#@overlay/match by=overlay.subset({"metadata":{"name":"istio-ingress"}})
+---
+spec:
+  servers:
+  #@overlay/append
+  - hosts:
+    - '*.apps.example.com'
+    port:
+      name: https-apps-example-com
+      number: 443
+      protocol: HTTPS
+    tls:
+      credentialName: wildcard-apps-example-com-cert
+      mode: SIMPLE
 ```
 
+This overlay can be applied during installation of
```

Comment:
@jenspinney @syerram we would like to know how you would expect operators to apply overlays?

Reply:
wish: a link to a generic process doc for "applying ytt overlays to cf-for-k8s deployments"

Reply:
Currently, the install script does not accept overlays parameters. FWIW, this looks ok to me.

In the long term, we will move away from the current install script and expect end-users to run `ytt | kapp` directly. We will provide documentation for operators on how they can add overlays to ytt (via -f)  among other things.

Reply:
Technically, the parameter that the script takes is passed to ytt as -f which can be either a file or a directory.

That's how we currently provide overlays and values to the script.